adal/src/main/java/com/microsoft/aad/adal/AuthenticationRequest.java

```diff
@@ -104,7 +106,21 @@
         mClaimsChallenge = claimsChallenge;
     }
 
-    AuthenticationRequest(String authority, String resource, String client, String redirect,
+    public AuthenticationRequest(String authority, String resource, String client, String redirect,
```

Comment:
can we see if we can prevent this explosion of constructors?

Reply:
since we need to make some change in the ad-accounts, I would keep the constructors in public-ADAL.